build.sbt

```diff
@@ -64,7 +64,8 @@ lazy val scoverageSettings: Seq[Setting[_]] = Seq(
     "metrics\\..*",
     "features\\..*",
     "test\\..*",
-    ".*(BuildInfo|Routes|Options|TestingUtilitiesController).*"
+    ".*(BuildInfo|Routes|Options|TestingUtilitiesController).*",
+    "logger.*\\(.*\\)"
```

Comment:
I tried to exclude logs from coverage, this line exclude just `logger.error(` If someone know how to make it to exclude message too please ping me

Reply:
nice! 👍